background.js

```diff
@@ -227,6 +216,34 @@ ipcMain.on('factory-reset', async () => {
   app.quit();
 });
 
+/**
+ * assume sync activities aren't going to be costly for a UI app.
+ * @param {String} name -- basename of the resource to link
+ * @param {Boolean} state -- true to symlink, false to delete
+ * @returns {null|Error}
```

Comment:
```suggestion
 * @returns {Promise<Error?>}
```
Because this is an `async function`.

Reply:
Done. Note that the checkboxes are still broken overall in this PR -- branch `80-get-checkboxes-working` will fix them.